### PR TITLE
Replace <br> with <br/>

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23827

## Description
Added translation from <br> to <br/> for compatibility with our docs site

### CODE REVIEW

File: demisto_sdk/commands/generate_docs/common.py [hunk 1]
```diff
@@ -196,9 +196,10 @@ def execute_command(command_example, insecure: bool):
             if entry.contents:
                 content = entry.contents
                 if isinstance(content, STRING_TYPES):
-                    md_example += content
+                    # The replacement of <br> by <br/> is for compatibility with our docs site
+                    md_example += content.replace('<br>', '<br/>')
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/common.py:200]: dont we also need to replace `<BR>` ?

Comment (ShahafBenYakir) [reply demisto_sdk/commands/generate_docs/common.py:200]: Is there anything like this ?!
will ask Francesco

File: demisto_sdk/commands/generate_docs/tests/generate_docs_test.py [hunk 2]
```diff
@@ -17,6 +17,24 @@
 # common tests
 
 
+def test_format_md():
+    from demisto_sdk.commands.generate_docs.common import format_md
+    md_returned_from_server = """
+    ### Domain List
+    |Malicious|Name|
+    |---|---|
+    | Vendor: HelloWorld<br>Description: Hello World returned reputation 88 | google.com |
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/tests/generate_docs_test.py:26]: lets check also `<BR>`

File: demisto_sdk/commands/generate_docs/tests/generate_docs_test.py [hunk 3]
```diff
@@ -17,6 +17,24 @@
 # common tests
 
 
+def test_format_md():
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/tests/generate_docs_test.py:20]: add test docstring `Given-When-Then`

File: demisto_sdk/commands/generate_docs/common.py [hunk 4]
```diff
@@ -256,6 +258,14 @@ def build_example_dict(command_examples: list, insecure: bool):
     return examples, errors
 
 
+def format_md(md: str):
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/common.py:261]: please add docstring

Comment (Itay4) [inline demisto_sdk/commands/generate_docs/common.py:261]:
```suggestion
def format_md(md: str) -> str:
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Added formatting to **generate-docs** command replacing all <br> tags with <br/>
```
Comment (Itay4) [inline CHANGELOG.md:2]:
```suggestion
* Added formatting to **generate-docs** command replacing all <br> tags with <br/>.
```

File: demisto_sdk/commands/generate_docs/common.py [hunk 6]
```diff
@@ -256,6 +258,19 @@ def build_example_dict(command_examples: list, insecure: bool):
     return examples, errors
 
 
+def format_md(md: str) -> str:
+    """
+    Formats a given md string by replacing <br> and <BR> tags with <br/>
+    :param md: string representing mark down
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/common.py:264]:
```suggestion
    Args:
        md (str): String representing mark down.
```

File: demisto_sdk/commands/generate_docs/common.py [hunk 7]
```diff
@@ -256,6 +258,19 @@ def build_example_dict(command_examples: list, insecure: bool):
...
+    :return: formatted string representing mark down
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/common.py:265]:
```suggestion
    Returns:
        str. Formatted string representing mark down.
```

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 8]
```diff
@@ -72,20 +72,20 @@ def generate_integration_doc(
 
         if not output:  # default output dir will be the dir of the input file
             output = os.path.dirname(os.path.realpath(input))
-        errors = []
+        errors: list = []
         example_dict = {}
-        if examples and os.path.isfile(examples):
+        if examples and os.path.isfile(examples):   # type: ignore
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:77]: why are we ignoring here?

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 9]
```diff
@@ -72,20 +72,20 @@ def generate_integration_doc(
...
             errors.extend(build_errors)
         else:
             errors.append(f'Command examples was not found {examples}.')
 
         if permissions == 'per-command':
-            command_permissions_dict = {}
-            if command_permissions and os.path.isfile(command_permissions):
+            command_permissions_dict: Any = {}
+            if command_permissions and os.path.isfile(command_permissions):     # type: ignore
                 command_permissions = get_command_permissions(command_permissions)
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:87]: why are we ignoring here?

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 10]
```diff
@@ -72,20 +72,20 @@ def generate_integration_doc(
...
-                for command_permission in command_permissions:
+                for command_permission in command_permissions:  # type: ignore
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:88]: why are we ignoring here?

File: demisto_sdk/commands/generate_docs/generate_playbook_doc.py [hunk 11]
```diff
@@ -117,9 +117,9 @@ def get_inputs(playbook: Dict) -> Tuple[List[Dict], List[str]]:
     inputs = []
 
     if not playbook.get('inputs'):
-        return {}, []
+        return [{}], []
 
-    for _input in playbook.get('inputs'):
+    for _input in playbook.get('inputs'):   # type: ignore
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_playbook_doc.py:122]: why are we ignoring here?

File: demisto_sdk/commands/generate_docs/generate_playbook_doc.py [hunk 12]
```diff
@@ -130,7 +130,7 @@ def get_inputs(playbook: Dict) -> Tuple[List[Dict], List[str]]:
         # is an Indicators Query input section
         if not name and isinstance(playbook_input_query, dict):
             name = 'Indicator Query'
-            _value = playbook_input_query.get('query')
+            _value = playbook_input_query.get('query')  # type: ignore
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_playbook_doc.py:133]: why are we ignoring here?

**Pull request merged**